sound/soc/sof/sof-audio.h

```diff
@@ -34,6 +34,10 @@
 #define SOF_PIN_TYPE_SINK	0
 #define SOF_PIN_TYPE_SOURCE	1
 
+/* Widget initialization blob style */
+#define SOF_INIT_BLOB_BASE_WITH_MOD_EXT	0
+#define SOF_INIT_BLOB_BASE_WITH_BASE_EXT	1
```

Comment:
well the values should be in the tokens.h definition, no? This defines the UAPI, it should be clear.
I am afraid we have similar issues with the pin type. Gah.